.github/workflows/publish-docs.yaml

```diff
@@ -2,10 +2,21 @@ name: Generate reference docs
 
 on:
   workflow_dispatch:
+    inputs:
+      open_pull_request:
+        description: 'Open a pull request against the docs repository'
+        required: false
+        default: 'false'
+        options:
+          - "true"
+          - "false"
```

Comment:
Is this for testing?

Reply:
Yeah I made this configurable in case we wanted to generate all the documentation and have them uploaded as assets to the run, but not actually open a pull request. Sort of a "dry run" functionality.

Reply:
If so can you leave a comment to explain how we expect this to be used?

Reply:
BTW I replied here before I saw your comment. I think it makes sense.